services/paybuttonService.ts

```diff
@@ -209,13 +209,11 @@ export async function createPaybutton (values: CreatePaybuttonInput): Promise<Pa
       include: includeAddresses
     })
     // Update cache for existent addreses
```

Comment:
Better comment would be explaining _when_ this happens.

Reply:
I think this comment was more useful before the creation of `CacheSet`; removed.

I don't understand what would be the _when_ you refer to, since this function will always update the cache

Reply:
Nevermind -- but comments should be useful if they're going to be there so I'll leave it up to you what you want to do with it.